.github/workflows/R-CMD-check.yaml

```diff
@@ -17,7 +17,8 @@ jobs:
         config:
           - {os: windows-latest, r: '3.6'}
           - {os: windows-latest, r: 'devel'}
-          - {os: macOS-latest, r: '3.6'}
+          - {os: macOS-latest, r: '3.6', dev_dplyr: true}
+          - {os: macOS-latest, r: '3.6', dev_dplyr: false}
```

Comment:
At this point I think we can have 3 variants of mac 3.6 for:

```
- {os: macOS-latest, r: '3.6', old_dplyr: true,  dev_tidymodels: true}
- {os: macOS-latest, r: '3.6', old_dplyr: true,  dev_tidymodels: false}
- {os: macOS-latest, r: '3.6', old_dplyr: false, dev_tidymodels: true}
- {os: macOS-latest, r: '3.6', old_dplyr: false, dev_tidymodels: false}
```

which later we can reduce to

```
- {os: macOS-latest, r: '3.6', old_dplyr: true}
- {os: macOS-latest, r: '3.6', old_dplyr: false}
```